packages/mint-components/src/components/sqm-coupon-code/sqm-coupon-code.feature

```diff
@@ -0,0 +1,120 @@
+@author:truman
+@owner:truman
+Feature: Coupon Code
+
+    The coupon code component is a box that allows users to see and copy their coupon code for a given program
+
+    @motivating
+    Scenario: A Users sharelink can be copied to their clipboard
+        Given tooltipText is "hello tooltip"
+        When the component renders
+        Then there is a textbox with the user's coupon code
+        When the clipboard icon is clicked
+        Then the link is copied to clipboard
+        And a tooltip appears for ~1 second
+
+    @motivating
+    Scenario: Component only looks at fueltank rewards
+        Given a user has multiple rewards of different types
+        When the component is loaded
+        Then it filters the user's reward to only return FUELTANK rewards
```

Comment:
We had talked about a program filter too, is this a thing?

Reply:
Added a new spec to address this